node/coinstacks/common/pulumi/src/api.ts

```diff
@@ -255,6 +256,9 @@ export async function deployApi(
               cpu: config.api.cpuLimit,
               memory: config.api.memoryLimit,
             },
+            requests: {
+              cpu: config.api.cpuRequest ?? config.api.cpuLimit,
+            },
```

Comment:
Another syntax for this would be:
```suggestion
            ...(config.api.cpuRequests && {
              requests: {
                cpu: config.api.cpuRequest,
              }
            }),
```
Which makes use of kube default of setting request to limit if not provided

Reply:
The other updates for this would be not require `cpuRequest` in `config.ts` check. And updating for the value to be optional in any config objects `cpuRequest?: string`